prototype/core/impl/src/main/java/org/eclipse/sensinact/prototype/whiteboard/impl/SensinactWhiteboard.java

```diff
@@ -508,7 +516,7 @@ public Promise<Object> act(String model, String provider, String service, String
         } else {
             Deferred<Object> d = promiseFactory.deferred();
             promiseFactory.executor().execute(() -> {
-                try {
+                try (IMetricTimer timer = metrics.withTimer("sensinact.whiteboard.act")) {
```

Comment:
I'm not sure that this is the correct way to measure this execution time. By measuring this way we lose the time between the call to `resolveWith` and the actual resolution if the action returns a promise.